nan_callbacks_12_inl.h

```diff
@@ -166,11 +166,11 @@ namespace imp {
 static
 void FunctionCallbackWrapper(const v8::FunctionCallbackInfo<v8::Value> &info) {
   v8::Local<v8::Object> obj = info.Data().As<v8::Object>();
-  FunctionWrapper *wrapper = static_cast<FunctionWrapper*>(
-      obj->GetAlignedPointerFromInternalField(kFunctionIndex));
+  FunctionCallback callback =  reinterpret_cast<FunctionCallback>(reinterpret_cast<intptr_t>(
```

Comment:
It can warn otherwise.

On August 27, 2015 7:22:29 PM EEST, Ben Noordhuis <notifications@github.com> wrote:
>> @@ -166,11 +166,11 @@ namespace imp {
>>  static
>>  void FunctionCallbackWrapper(const
>v8::FunctionCallbackInfo<v8::Value> &info) {
>>    v8::Local<v8::Object> obj = info.Data().As<v8::Object>();
>> -  FunctionWrapper *wrapper = static_cast<FunctionWrapper*>(
>> -      obj->GetAlignedPointerFromInternalField(kFunctionIndex));
>> +  FunctionCallback callback = 
>reinterpret_cast<FunctionCallback>(reinterpret_cast<intptr_t>(
>
>I don't think you need the extra intptr_t cast.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/nodejs/nan/pull/431/files#r38115120